packages/account/src/Helpers/utils.ts

```diff
@@ -183,6 +62,129 @@ export const shouldShowIdentityInformation = ({
 };
 
 export const getDocumentData = (country_code: string, document_type: string) => {
+    // Note: Ensure that the object keys matches BE API's keys. This is simply a mapping for FE templates
+    const idv_document_data = Object.freeze({
```

Comment:
Do we need this change? isnt it old approach better? we can export the const if we need

Reply:
+1

Reply:
and if we are declaring inside the function, we don't need to freeze it as it's scoped

Reply:
Objects dont trigger localize() when language is changed in platform. Hence converted this into a function

Reply:
yep. My question is can we declare it as separate function?